pkg/utils/metrics/metrics.go

```diff
@@ -0,0 +1,102 @@
+/*
+metrics tracks request behavior inside tenant (which manages Ensign user accounts,
+projects, and topics) and quarterdeck (which manages authentication and authorization)
+*/
+package metrics
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/rs/zerolog/log"
+)
+
+// Prometheus namespaces for the collectors defined in this package.
+const (
+	NamespaceHTTPMetrics = "http_stats"
+)
+
+var (
+	// Active HTTP requests
+	ActiveTenantRequests prometheus.Gauge
+	ActiveQDRequests     prometheus.Gauge
+
+	// Total HTTP requests, disaggregated by service (e.g. "tenant" or "quarterdeck", status code, query path)
+	RequestsHandled *prometheus.CounterVec
+
+	// HTTP request duration, disaggregated by service (e.g. "tenant" or "quarterdeck", status code, query path)
+	RequestDuration *prometheus.HistogramVec
+
+	// Protection from unintentionally re-registering collectors
+	setup sync.Once
+	err   error
+)
+
+func Serve() {
```

Comment:
```suggestion
func Setup() {
```

I think `metrics.Setup()` might be more informative.